src/parser/parser_v2.cpp

```diff
@@ -136,12 +165,18 @@ condition::ptr parse_rule_condition(const parameter::map &root, manifest &target
         target.root = target_manifest.insert(address);
         target.name = address;
         target.key_path = std::move(kp);
+        auto input_transformers = at<parameter::vector>(input, "transformers", {});
+        if (input_transformers.empty()) {
+            target.transformers = transformers;
+        } else {
+            target.transformers = parse_transformers(input_transformers, source);
```

Comment:
You don't seem to be handling `source` correctly. FWICT `parse_transformers` takes the source by reference, so it will change the default for subsequent inputs. It will also affect the value used when creating the `condition`.